draft-ietf-rats-corim.md

```diff
@@ -111,59 +109,117 @@ informative:
+The Verifier may produce new assertions as a result of an applied appraisal policy.
+For example, if an appraisal procedure finds all of the components of a subsystem are configured correctly, the policy may direct the Verifier to produce new assertions, "Subsystem=X" has the Claim "TRUSTED=TRUE".
+Consequently, the internal ACS structure is a reconciled conversation between several producers of RATS Conceptual Messages that has mapped each message into a consistent internal representation, has associated the identity of the corresponding RATS role with each assertion (the authority), and has applied Conceptual Message constraints to the assertion.
+
+The CoRIM data model specified in this document covers the RATS Conceptual Message types, `Reference Values` and `Endorsements`.
+Reference values and Endorsements are required for Verifier reconciliation (and Evidence is required for corresponding internal ACS creation) as illustrated in {{sec-interact-acs}}.
+
+## Internal Representation {#sec-internal-rep}
+
+In this document CDDL is used to specify both the CoRIM structure and to specify an internal representation for use in the appraisal procedure.
+The actual internal representation of a Verifier is implementation-specific and out-of-scope of this document.
+Requirements for an internal representation of Conceptual Messages are defined in {{tbl-cmrr}}, where each Conceptual Message type has a structure as depicted by the *Structure* column.
+The internal representations used by this document are defined in {{sec-ir-cm}}.
+
+## Interacting with an ACS {#sec-interact-acs}
+
+Conceptual Messages interact with an ACS by specifying criteria that should be met by the ACS and by presenting the assertions that should be added to the ACS if the criteria are satisfied.
+Internal representations of Conceptual Messages, ACS, and ARS should satisfy the following requirements for Verifier reconciliation and appraisal processing:
+
+| CM Type | Structure | Description |
+|---
+| Evidence | List of Evidence claims | If the Attester is authenticated, add Evidence claims to the ACS with Attester authority |
+| Reference Values | List of Reference Values claims | If a reference value in a CoRIM matches claims in the ACS, then the authority of the CoRIM issuer is added to those claims. |
+| Endorsements | List of expected actual state claims, List of Endorsed Values claims | If the list of expected claims are in the ACS, then add the list of Endorsed Values claims to the ACS with Endorser authority |
+| Series Endorsements | List of expected actual state claims, Series of tuples containing selection and series Endorsement claims | If the list of expected claims are in the ACS, and if the series selection condition is satisfied, then add the series Endorsement claims for the first series tuple with satisfied condition to the ACS with Endorser authority |
```

Comment:
unclear what "selection" means in this context

Reply:
It is a second stage condition. 
The series structure is a list of condition-addition 2-tuples. Selection is another word for condition that isn't confused with the endorsement's first-stage condition.
Possibly, the second sentence here is a clearer explanation of selection. The current text assumes the reader read ahead to understand it's use here.

Reply:
OK, thanks. The problem is this is not self-contained.  At least there should be a forward reference.